en/Help and support.md

```diff
@@ -2,7 +2,11 @@ Learn how you can get help and support resources when using Obsidian or any of o
 
 ## Contact Obsidian support
 
-For questions and other support related to our paid services, such as [[Introduction to Obsidian Publish|Obsidian Publish]] and [[Introduction to Obsidian Sync|Obsidian Sync]], you can [email the Obsidian team](mailto:support@obsidian.md).
+If you have question of need help with any of the following, you can [email the Obsidian team](mailto:support@obsidian.md) for support:
+
+- Account and billing support
+- Support related to our paid services, such as [[Introduction to Obsidian Publish|Obsidian Publish]] and [[Introduction to Obsidian Sync|Obsidian Sync]]
+- Support for users with [[Catalyst license]] or [[Commercial license]]
```

Comment:
Sorry, I meant support related to their catalyst licenses or commercial licenses, for example update name of their commercial license etc.

To be clear, we don't provide technical support for "my obsidian isn't working" type of inquiries, those go to the community.

Sorry I wasn't being clear!

Reply:
Done! ✅